src/components/UniversalNavbarExpanded/MobileNav/MobileNav.js

```diff
@@ -78,6 +79,7 @@ const MobileNav = ({
   hideMobileCta,
   ctaButtonTrackingFunction,
   LinkComponent,
+  singleCta,
```

Comment:
```suggestion
  singleCta = {},
```
 ah, better do it here too, sorry i missed this before